care/users/tests/test_change_password.py

```
@@ -1,51 +1,49 @@
+        self.payload["new_password"] = "123"
+        response = self.client.put(self.url, self.payload, format="json")
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertContains(
+            response,
+            "This password is too short",
+            status_code=status.HTTP_400_BAD_REQUEST,
+        )
 
-        # Verify the password actually changed in the DB
+    def test_change_password_invalid_password(self):
+        self.payload["new_password"] = "password123"
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to: "new_password"

(S105)